backend/app/api/endpoints/projects.py

```
@@ -0,0 +1,241 @@
+            detail=f"Failed to remove task from project: {str(e)}",
+        )
+
+
+@router.put(
+    "/{project_id}/tasks/reorder",
+    response_model=list[ProjectTaskResponse],
+)
+def reorder_project_tasks_endpoint(
+    project_id: int = Path(..., description="Project ID"),
+    reorder_data: ProjectTaskReorderRequest = None,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)